content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Kops
+description: Instructions to setup Kops for use with Istio.
+weight: 20
```

Comment:
I haven't looked at weights, but believe this might be the same as another page and thus the order is indeterminate. In the past, multiple pages might need to have weights updated to keep the order.

Reply:
The intent is to put them in alphabetical order, so it should come right after the `kind` one. We should probably do a cleanup to renumber the weights of all the entries starting with 10, 20, 30, ... (so we can easily add new ones in between existing entries) or ignore weights and have the site generator sort them alphabetically during the build. For now I suggest changing the weight to 17, which will probably put it in the right spot (or close), and then we can fix the weights in a future PR.
```suggestion
weight: 17
```

Reply:
I disagree with Frank and think we need to force an update in weights in this PR (or maybe we write a precursor PR to open up space for this one (and I may do that very quickly)). But we can't just go overlap another weight.

Currently, it's GKE:15, IBM:16, kind:17, Gardener:18, kubesphere:19, and microk8s:20 so there is no room (ie, no 17.5). We either need to move 3 pages up or 3 pages down (and maybe more).

Reply:
I agree that we should fix this properly, but I think 17 puts it in the right spot, just by luck, so there's no urgency to fix it. We can fix it after this PR. See my other comment (about old versions). That is the bigger issues with this entry IMO.

Reply:
https://github.com/istio/istio.io/pull/6747 opened to renumber. If that's accepted, this one could be 32 or 33.